packages/core/src/__tests__/initialize.test.ts

```diff
@@ -158,4 +161,39 @@ describe("iOS: initialize", () => {
     expect(mockStart).not.toHaveBeenCalled();
     expect(isStarted).toBe(true);
   });
+
+  test("sdk not already started, missing app id", async () => {
```

Comment:
these two were moved because they were under the "Android: initialize" suite previously but are testing ios functionality